src/page-multi-facility/MultiFacilityImpactDashboard.tsx

```diff
@@ -118,7 +118,7 @@ const MultiFacilityImpactDashboard: React.FC<Props> = ({
           facilities?.data.map((facility, index) => {
             return (
               <EpidemicModelProvider
-                key={index}
+                key={facility.id}
```

Comment:
Just want to 100% verify, though I'm sure you tested this locally: the facility id is _not_ the facility name, right? If I have multiple facility models with the same name, this still works as expected?

Reply:
yes it's a random database ID string